backend/src/resolvers/bill.ts

```diff
@@ -1,22 +1,22 @@
 import { get } from 'http';
-import { CreateBill, DeleteBill, EditBill, GetBills } from '../handlers/billHandler';
+import * as billHandler from '../handlers/billHandler';
 import { GetAllocations } from '../handlers/allocationHandler';
 
 export const resolvers = {
     Query: {
-        bills: async (_: any, { houseid }: any) => {
-            return await GetBills(houseid);
+        bills: async (_: any, { houseid }: {houseid: String}) => {
+            return await billHandler.GetBills(houseid);
```

Comment:
The await keyword is unnecessary when directly returning a promise, you can remove await and just return the promise.
 
 This applies to all resolvers